spec/features/index_displays_station_info_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../spec_helper'
+
+describe "When user visits index" do
+  describe "The user sees all of the data" do
+    it "for one station " do
+      Station.create(name: "Station Name",
+                    city_id: City.find_or_create_by(name: "Denver").id,
+                    dock_count_id: DockCount.find_or_create_by(dock_number: 15).id,
+                    installation_date_id: InstallationDate.find_or_create_by(installation_date: 20100102).id)
+
+      visit('/stations')
+
+      within("h1") do
+        expect page.has_content?("Stations")
+      end
+      within("h3") do
+        expect page.has_content?("Station Name")
+      end
+      within(".city") do
+        expect page.has_content?("Denver")
+      end
+    end
+  end
+
+  describe "Has link" do
```

Comment:
I would put the "Has link" and "has button" assertions into the test above. Should keep you from having to create a station three times, which can get expensive as the test suite grows.